cli/cobra.go

```diff
@@ -240,16 +246,51 @@ func hasManagementSubCommands(cmd *cobra.Command) bool {
 	return len(managementSubCommands(cmd)) > 0
 }
 
+func hasOrchestratorSubCommands(cmd *cobra.Command) bool {
```

Comment:
with Kubernetes support removed, maybe be more explicit with `hasSwarmSubCommands`?